src/views/certifications/components/signup-form-area/index.tsx

```diff
@@ -11,12 +11,16 @@ import {
 	SubmitMessage,
 } from './components/newsletter-signup-form'
 import s from './signup-form-area.module.css'
+import { HTMLAttributes } from 'react'
+import classNames from 'classnames'
 
 const APPEARANCE_SLUG = 'dark'
 
-export function SignupFormArea() {
+export function SignupFormArea({
+	className = '',
```

Comment:
No default value needed here
```suggestion
	className
```